tests/jballerina-unit-test/src/test/resources/test-src/record/cyclic_record_via_fields.bal

```diff
@@ -0,0 +1,9 @@
+public type ClientConfiguration record {|
```

Comment:
Shall we add a project with the two files like in https://github.com/ballerina-platform/ballerina-lang/issues/30347#issuecomment-832030912 and test instead? I was unable to reproduce this issue with a single bal file.

Also missing the licence header.

Reply:
Sure, I will add. I was able to reproduce the issue with this perticular test case.

Reply:
Oh okay, if you were able to reproduce it with this let's keep this test also.